android/src/main/java/com/mattermost/networkclient/APIClientModule.kt

```diff
@@ -26,6 +28,9 @@ class APIClientModule(reactContext: ReactApplicationContext) : ReactContextBaseJ
             // Attach client options if they are passed in
             sessionsClient[baseUrl]!!.parseOptions(options, sessionsRequest[baseUrl]);
 
+            // Protocols
+            sessionsClient[baseUrl]!!.protocols(listOf(Protocol.HTTP_1_1))
```

Comment:
Why 1.1? Will this work with http2?

Reply:
Whoops! Was testing against various servers on my local - removed.